doc/loganalyzer_to_pytest.md

```diff
@@ -0,0 +1,74 @@
+#### Requirements
+Move existed loganalyzer to be supported by pytest framework.
+
+#### Overview
+At this approach all the work can be done on the Ansible host if script will download extracted syslog file to the Ansible host.
+
+Current loganalyzer implementation description:
+https://github.com/Azure/SONiC/wiki/LogAnalyzer
+
+#### Development
+Module "loganalyzer.py" with class "Loganalyzer". Class is intendent to do:
+- Add start marker to the DUT syslog
+- Extract DUT syslog based on the markers. Download extracted logs to the Ansible host
+- Perform analize of downloaded syslog and return the result
+
+- Configure regular expressions which will be used to found matches in the logs
+
+"Loganalyzer" class interface:
+- __init__(ansible_host: ansible_host)
+- load_common_config() - Load regular expressions from common configuration files: match, expect and ignore which are located in some configuration directory or with current module. Clear previous configured match, expect and ignore. Save loaded configuration.
+- parse_regexp_file(file_path) - read and parse regular expressions from specified file. Return list of strings of defined regular expressions.
+- run_cmd(callable, *args, **kwargs) - call function and analyze DUT logs, return the same result as "analyze" function
+- init() - Add start marker to the DUT syslog.
+- analyze() - Extract syslog and copy it to ansible host. Analyze extracted syslog files localy. Return python dictionary object.

```

Comment:
Extract syslog and copy it to sonic-mgmt container?

Reply:
Logs will be stored to the host where pytest was started. For now it will be sonic-mgmt container. It will decrease work which are doing on the device side.